src/EventStore.ClusterNode/Program.cs

```diff
@@ -96,8 +96,7 @@ internal static class Program {
 				if (options.Unknown.Options.Any() && !options.Application.AllowUnknownOptions) {
 					Log.Fatal(
 						$"Found unknown options. To continue anyway, set {nameof(ClusterVNodeOptions.ApplicationOptions.AllowUnknownOptions)} to true.");
-					Log.Information($"Options:{Environment.NewLine}{ClusterVNodeOptions.HelpText}");
-
+					Log.Information("Use --help parameter in the command line to see full list of EventStoreDB configuration options.");
```

Comment:
I think that a better way to word this would be:

```suggestion
					Log.Information("Use the --help option in the command line to see the full list of EventStoreDB configuration options.");
```